sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ### 4.6.1 (Unreleased)
 
 #### Features Added
+* Adds vector embedding policy and vector indexing policy. See [PR 34882](https://github.com/Azure/azure-sdk-for-python/pull/34882).
+* Adds support for vector search non-streaming order by queries. See [PR 10101](https://github.com/Azure/azure-sdk-for-python/pull/10101).
```

Comment:
The PR referenced here seems wrong

Reply:
YES lol it was a place holder, thank you for catching that!